src/Uno.UI.Tests/Windows_UI_XAML_Controls/GridTests/Given_Grid.cs

```diff
@@ -23,142 +15,174 @@ public class Given_Grid : Context
 
-			SUT.Measure(new Windows.Foundation.Size(20, 20));
+			SUT.Measure(new Size(20, 20));
 			var measuredSize = SUT.DesiredSize;
-			SUT.Arrange(new Windows.Foundation.Rect(0, 0, 20, 20));
+			SUT.Arrange(new Rect(0, 0, 20, 20));
 
-			Assert.AreEqual(new Windows.Foundation.Rect(0, 0, 20, 20), SUT.GetChildren().First().Arranged);
+			Assert.AreEqual(new Rect(0, 0, 20, 20), SUT.GetChildren().First().Arranged);
 
-			Assert.AreEqual(measuredSize, new Windows.Foundation.Size(10, 10));
+			Assert.AreEqual(measuredSize, new Size(10, 10));
 			Assert.AreEqual(1, SUT.GetChildren().Count());
 		}
 
 		[TestMethod]
 		public void When_Grid_Has_One_Element_And_MinWidth_MinHeight_VerticalAlignment_Top_HorizontalAlignment_Center()
 		{
-			var SUT = new Grid() { Name = "test" };
+			var SUT = new Grid { Name = "test" };
 
 			SUT.MinWidth = 40;
 			SUT.MinHeight = 40;
 			SUT.VerticalAlignment = VerticalAlignment.Top;
 			SUT.HorizontalAlignment = HorizontalAlignment.Center;
 
-			SUT.AddChild(new View { Name = "Child01", RequestedDesiredSize = new Windows.Foundation.Size(20, 20) });
+			SUT.AddChild(new View {Name = "Child01", RequestedDesiredSize = new Size(20, 20)});
 
-			SUT.Measure(new Windows.Foundation.Size(60, 60));
+			SUT.Measure(new Size(60, 60));
 			var measuredSize = SUT.DesiredSize;
-			SUT.Arrange(new Windows.Foundation.Rect(0, 0, 60, 60));
+			SUT.Arrange(new Rect(0, 0, 60, 60));
 
-			Assert.AreEqual(new Windows.Foundation.Rect(0, 0, 40, 40), SUT.GetChildren().First().Arranged);
+			Assert.AreEqual(new Rect(0, 0, 40, 40), SUT.GetChildren().First().Arranged);
 
-			Assert.AreEqual(measuredSize, new Windows.Foundation.Size(40, 40));
+			Assert.AreEqual(measuredSize, new Size(40, 40));
 			Assert.AreEqual(1, SUT.GetChildren().Count());
 		}
 
 		[TestMethod]
 		public void When_Grid_Has_Two_Elements_And_MinWidth_MinHeight_VerticalAlignment_Top_HorizontalAlignment_Center_And_Child_Stretched_And_Centered()
 		{
-			var SUT = new Grid() { Name = "test" };
+			var SUT = new Grid { Name = "test" };
 
 			SUT.MinWidth = 40;
 			SUT.MinHeight = 40;
 			SUT.VerticalAlignment = VerticalAlignment.Top;
 			SUT.HorizontalAlignment = HorizontalAlignment.Center;
 
-			var c1 = SUT.AddChild(new View { Name = "Child01", RequestedDesiredSize = new Windows.Foundation.Size(20, 20), HorizontalAlignment = HorizontalAlignment.Stretch, VerticalAlignment = VerticalAlignment.Stretch });
-			var c2 = SUT.AddChild(new View { Name = "Child02", RequestedDesiredSize = new Windows.Foundation.Size(20, 20), HorizontalAlignment = HorizontalAlignment.Center, VerticalAlignment = VerticalAlignment.Center });
+			var c1 = SUT
+				.AddChild(new View
+				{
+					Name = "Child01",
+					RequestedDesiredSize = new Size(20, 20),
+					HorizontalAlignment = HorizontalAlignment.Stretch,
+					VerticalAlignment = VerticalAlignment.Stretch
+				});
+			var c2 = SUT
+				.AddChild(new View
+				{
+					Name = "Child02",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'Child02' 17 times.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826507)